include/arch/x86/arch/model/statedata.h

```diff
@@ -81,7 +81,7 @@ extern uint32_t x86KSFirstValidIODomain;
 #ifdef CONFIG_PRINTING
 extern uint16_t x86KSconsolePort;
```

Comment:
Loosely related, but I wonder if `x86KSconsolePort` use anywhere actually? It seems only `x86KSdebugPort` is used for printing in the end, see https://github.com/seL4/seL4/blob/master/src/plat/pc99/machine/io.c

Reply:
It's used by the multiboot cmdline parsing in cmdline.c to save the console_port option in.